tests/cppException/enc/enc.cpp

```diff
@@ -5,47 +5,31 @@
 #include <openenclave/enclave.h>
 #include <openenclave/internal/atexit.h>
 #include <openenclave/internal/print.h>
-#include "../args.h"
+#include "cppException_t.h"
 
 bool TestCppException();
 
-OE_ECALL void Test(void* args_)
+extern "C" int test(void)
```

Comment:
Can you retain the function name as Test unless this is forbidden by the naming convention?

Reply:
The Development Guide states that the Naming Convention is lower_camel_case.  I have used the lower_camel_case naming convention in the EDL file in order to comply with the naming convention.

Reply:
Thanks.